addons/bus/models/bus_presence.py

```diff
@@ -65,11 +65,19 @@ def _update_presence(self, inactivity_period, identity_field, identity_value):
         if not presence:  # create a new presence for the user
             values[identity_field] = identity_value
             values['last_presence'] = last_presence
+            values["status"] = "online"
             self.create(values)
         else:  # update the last_presence if necessary, and write values
             if presence.last_presence < last_presence:
                 values['last_presence'] = last_presence
```

Comment:
What is this condition exactly? It looks like we should just write what we receive now (but I guess this field becomes obsolete now).

Reply:
There is no point in going backwards. If a device says it was last active at say 06:00, and another says 05:30 we want to keep the most recent one.